x/slashing/keeper/infractions.go

```diff
@@ -44,7 +45,7 @@ func (k Keeper) HandleValidatorSignature(ctx sdk.Context, addr cryptotypes.Addre
 		panic(errors.Wrap(err, "failed to get the validator's bitmap value"))
 	}
 
-	missed := !signed
+	missed := signed == comet.BlockIDFlagAbsent
```

Comment:
if a validator votes no on a block they did sign so we only count misses when no vote is present. this is the same as before